audit/projects/k8s-conform/buckets/k8s-conform-capi-openstack/iam.json

```diff
@@ -15,6 +15,12 @@
       ],
       "role": "roles/storage.legacyBucketReader"
     },
+    {
+      "members": [
+        "group:k8s-infra-conform-capi-openstack@kubernetes.io"
```

Comment:
This seems to be correct, but I don't know why it wasn't there before.  It looks like it was not run when it was added.

@dims - I see in the logs that you manually created service accounts in this project and then generated and downloaded credentials for them (Feb 26).  Can you explain this?

First, we REALLY don't want to be handing out long-lived credentials like that.

Second, we REALLY REALLY do not want to be making manual changes that are not backed by a script.

Third, we REALLY REALLY REALLY don't want people making unilateral changes with no history.  Was there history here?

Reply:
@thockin logged an issue here when it was done:
https://github.com/kubernetes/k8s.io/issues/604

more context in:
https://github.com/kubernetes/k8s.io/pull/459

Reply:
cool, thanks!  I will ask questions there.